runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -273,6 +273,11 @@
          */
         int checkpointRetries = 5;
 
+        /*
+         * The maximum number of SMR entries that will be grouped in a CheckpointEntry.CONTINUATION
+         */
+        int checkpointBatchSize = 50;
```

Comment:
did we come up with the number 50 empirically or some other strategy?

Reply:
Originally it was set to 50 (empirically based on entry size and 25mb write size limit) until a few months back we removed this parameter. We need it because without it if the compression ratio is high then we end up batching too many entries.

Reply:
ok